external_services/lti.py

```diff
@@ -63,7 +63,10 @@ def __init__(self, service, user, instance, request, title, context_id=None, lin
             "launch_presentation_return_url": request.scheme + '://' + request.get_host() + instance.get_absolute_url(),
 
             "tool_consumer_instance_guid": request.get_host() + "/aplus",
-            "tool_consumer_instance_name": "A+ LMS",
+            "tool_consumer_instance_name": settings.BRAND_NAME + " LMS",
```

Comment:
I don't like adding `LMS` here. If `A+` would be too simple, then we should add another settings variable. Consider case where BRAND_NAME is `Plussa`, wihich would render `Plussa LMS` here, and that is not a thing.

Reply:
I didn't really like it either, I just wasn't sure if "A+" is enough. Is it ok if it's just "A+"?

Reply:
Let's go with just the BRAND_NAME. If that is not suitable, we can change it later.